docs/book/src/plugins/creating-plugins.md

```diff
@@ -25,54 +25,93 @@ If you do not want to develop your plugin using Golang, you can follow its stand
+- [Plugin phase 2.0][plugin-2.0]: allow the Kubebuilder CLI or any other CLI, which is [Extending the Kubebuilder CLI][extending-cli], to discover external plugins, in this way, allow the users to use these external options as helpers to perform the scaffolds with the tool.
 - [Config-gen][config-gen]: the config-gen option has been provided as an alpha option in the Kubebuilder CLI(`kubebuilder alpha config-gen`) to encourage its contributions. The idea of this option would simplify the config scaffold. For further information see its [README][config-gen-readme].
-- [New Plugin (`deploy-image.go.kubebuilder.io/v1beta1`) to generate code][new-plugin-gen]: its purpose is to provide an arch-type that will scaffold the APIs and Controllers with the required code to deploy and manage solutions on the cluster. 
-
-Please, feel to contribute with them as well. Your contribution to the project is very welcome.  
-
-[sdk-cli-pkg]: https://github.com/operator-framework/operator-sdk/blob/master/internal/cmd/operator-sdk/cli/cli.go
-[sdk-ansible]: https://github.com/operator-framework/operator-sdk/tree/master/internal/plugins/ansible/v1
-[sdk-helm]: https://github.com/operator-framework/operator-sdk/tree/master/internal/plugins/helm/v1
-[operator-pattern]: https://kubernetes.io/docs/concepts/extend-kubernetes/operator/
-[controller-runtime]: https://github.com/kubernetes-sigs/controller-runtime
-[plugin-2.0]: https://github.com/kubernetes-sigs/kubebuilder/issues/1378
-[config-gen-readme]: https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/cli/alpha/config-gen/README.md
-[config-gen]: https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/cli/alpha/config-gen
-[plugins-phase1-design-doc-1.5]: https://github.com/kubernetes-sigs/kubebuilder/blob/master/designs/extensible-cli-and-scaffolding-plugins-phase-1-5.md
-[extending-cli]: extending-cli.md
-[new-plugin-gen]: https://github.com/kubernetes-sigs/kubebuilder/blob/master/designs/code-generate-image-plugin.md
-[kubebuilder-declarative-pattern]: https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern
-[declarative-code]: https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugins/golang/declarative
-[sdk]: https://github.com/operator-framework/operator-sdk
\ No newline at end of file
+- [New Plugin (`deploy-image.go.kubebuilder.io/v1beta1`) to generate code][new-plugin-gen]: its purpose is to provide an arch-type that will scaffold the APIs and Controllers with the required code to deploy and manage solutions on the cluster.
+
+Please, feel to contribute with them as well. Your contribution to the project is very welcome.
+
+- [sdk-cli-pkg](https://github.com/operator-framework/operator-sdk/blob/master/internal/cmd/operator-sdk/cli/cli.go)
```

Comment:
Seems like the following links are missing in the web page: https://book.kubebuilder.io/plugins/creating-plugins.html#future-vision-for-kubebuilder-plugins

Do we want to display them? Just convert them to be visible as a list:
https://deploy-preview-2934--kubebuilder.netlify.app/plugins/creating-plugins.html#future-vision-for-kubebuilder-plugins

How would you like this?

Reply:
Sounds good! Yeah, I like the idea of a new KubeGen Project. I'll remove this entire section now.